can/logger.py

```diff
@@ -213,13 +213,13 @@ def main() -> None:
     print(f"Connected to {bus.__class__.__name__}: {bus.channel_info}")
     print(f"Can Logger (Started on {datetime.now()})")
 
-    options = {"append": results.append}
     if results.file_size:
+        options = {"append": results.append}
```

Comment:
This seems out of scope for a ASC related PR

Reply:
Yes it is. There was an error when `-a` was used without `-s`, so this was a quick fix for that. #1361 resolved the issue. I need to revert the commit that introduced this.